manifests/site.pp

```diff
@@ -148,6 +148,12 @@
   include role::jenkins::master
 }
 
+node 'trusted.ci.jenkins.io' {
+  $hiera_role = 'trustedci'
+  sshkeyman::hostkey { ['trusted.ci.jenkins.io', 'ci.trusted.jenkins.io']: }
```

Comment:
What's the second hostkey for here? Are you planning to put more under "trusted.jenkins.io" in the future?

Reply:
It was a blind stupid copy/paste from above :D written by a tall guy :D